queue_services/entity-filer/src/entity_filer/filing_processors/dissolution.py

```diff
@@ -66,6 +66,7 @@ def process(business: Business, filing: Dict, filing_rec: Filing, filing_meta: F
 
     business.state = Business.State.HISTORICAL
     business.state_filing_id = filing_rec.id
+    business.restoration_expiry_date = None
```

Comment:
replicate the same code in https://github.com/bcgov/lear/blob/main/queue_services/business-filer/src/business_filer/filing_processors/dissolution.py#L77